panther_analysis_tool/main.py

```diff
@@ -262,73 +263,127 @@ def test_analysis(args: argparse.Namespace) -> Tuple[int, list]:
         A tuple of the return code, and a list of tuples containing invalid specs and their error.
     """
     failed_tests: DefaultDict[str, list] = defaultdict(list)
-    tests: List[str] = []
     logging.info('Testing analysis packs in %s\n', args.path)
 
-    # First classify each file
-    global_analysis, analysis, invalid_specs = classify_analysis(
+    # First classify each file, always include globals and data models location
+    data_models, global_analysis, analysis, invalid_specs = classify_analysis(
         list(load_analysis_specs(args.path)) +
-        list(load_analysis_specs(HELPERS_LOCATION)))
+        list(load_analysis_specs(HELPERS_LOCATION)) +
+        list(load_analysis_specs(DATA_MODEL_LOCATION)))
 
-    if len(analysis) == 0 and len(global_analysis) == 0:
+    if all(len(x) == 0 for x in [data_models, global_analysis, analysis]):
```

Comment:
Just realized that if all the specification files are invalid, this will return `Nothing to test` instead of `Invalid analysis`. I know it's not necessarily related to this change, but do you think it would be reasonable to add something like this

```
    if all(len(x) == 0 for x in [data_models, global_analysis, analysis]):
          if len(invalid_specs) > 0:
                return 1, invalid_specs
          return 1, ["Nothing to test in {}".format(args.path)]
```
If it's too outside the scope of this PR we can add it in another one.

Reply:
Ohh, good point! Seems like a small enough change to include here; I'll add it.